Tests/Unit/CollectionExtensionsTests.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace LoRaWan.Tests.Unit.LoRaWanTests
+namespace LoRaTools.Tests.Unit.Utils
```

Comment:
[nit] Why have a separate namespace just for this class? Should we consider to parent it under `LoRaTools.Tests.Unit`? That would make it easier to navigate in the test explorer

Reply:
Actually `CollectionExtensions` is part of `LoRaTools.Utils`, so the test should be in namespace `LoRaWan.Tests.Unit.LoRaTools.Utils` and located in the `Tests/Unit/LoRaTools/` folder, changed accordingly.